packages/webapp/pages/squads/discover/featured.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { ReactElement } from 'react';
+import { NextSeoProps } from 'next-seo/lib/types';
+import { NextSeo } from 'next-seo';
+import { SOURCES_QUERY } from '@dailydotdev/shared/src/graphql/squads';
+import InfiniteScrolling, {
+  checkFetchMore,
+} from '@dailydotdev/shared/src/components/containers/InfiniteScrolling';
+import { InfiniteData, useInfiniteQuery } from '@tanstack/react-query';
+import { ClientError } from 'graphql-request';
+import { FeedContainer, SquadGrid } from '@dailydotdev/shared/src/components';
+import { GetStaticPropsResult } from 'next';
+import { ApiError } from 'next/dist/server/api-utils';
+import { oneHour } from '@dailydotdev/shared/src/lib/dateFormat';
+import { gqlClient } from '@dailydotdev/shared/src/graphql/common';
+import { Squad } from '@dailydotdev/shared/src/graphql/sources';
+import CustomAuthBanner from '@dailydotdev/shared/src/components/auth/CustomAuthBanner';
+import { StaleTime } from '@dailydotdev/shared/src/lib/query';
+import { SourcesQueryData } from '@dailydotdev/shared/src/hooks/source/useSources';
+import { useViewSize, ViewSize } from '@dailydotdev/shared/src/hooks';
+import { SquadList } from '@dailydotdev/shared/src/components/cards/squad/SquadList';
+import { SquadDirectoryLayout } from '../../../../shared/src/components/squads/layout/SquadDirectoryLayout';
+import { getTemplatedTitle } from '../../../components/layouts/utils';
+import { defaultOpenGraph, defaultSeo } from '../../../next-seo';
+import { getLayout } from '../../../components/layouts/FeedLayout';
+import { mainFeedLayoutProps } from '../../../components/layouts/MainFeedPage';
+
+export type Props = {
+  initialData?: InfiniteData<SourcesQueryData<Squad>>;
+};
+
+const seo: NextSeoProps = {
+  title: getTemplatedTitle('Squad directory'),
+  openGraph: { ...defaultOpenGraph },
+  ...defaultSeo,
+};
+
+const SquadsPage = ({ initialData }: Props): ReactElement => {
+  const queryResult = useInfiniteQuery(
+    ['sourcesFeed'],
+    ({ pageParam }) =>
+      gqlClient.request(SOURCES_QUERY, {
+        filterOpenSquads: true,
+        featured: true,
+        first: 100,
+        after: pageParam,
+      }),
+    {
+      getNextPageParam: (lastPage) =>
+        lastPage?.sources?.pageInfo?.hasNextPage &&
+        lastPage?.sources?.pageInfo?.endCursor,
+      initialData,
+      staleTime: StaleTime.Default,
+    },
+  );
+  const isTablet = useViewSize(ViewSize.Tablet);
+  const flatSquads = queryResult?.data?.pages?.flatMap(
+    (page) => page.sources.edges,
+  );
+
+  return (
+    <>
+      <NextSeo {...seo} />
+
+      <SquadDirectoryLayout>
+        {isTablet ? (
+          <InfiniteScrolling
```

Comment:
Why is the infinite scrolling only for tablet?
How does the other one load the rest then

Reply:
Fixed! @rebelchris